src/main/resources/application-custom.yaml

```diff
@@ -186,8 +186,8 @@ hapi:
     binary_storage_enabled: true
     bulk_export_enabled: true
     subscription:
-      resthook_enabled: ${subscription.resthook.enabled}
-      websocket_enabled: ${subscription.websocket.enabled}
+      resthook_enabled: true
```

Comment:
Do we actually use either of these capabilities?

Reply:
I remeber very initially when we stand up the fhir server, Aziz asked to enable this capabilities, I think it might be used.